src/page/CheckRouter.test.tsx

```diff
@@ -9,6 +11,14 @@ import { PLUGIN_URL_PATH } from 'components/Routing.consts';
 import { CheckRouter } from './CheckRouter';
 
 describe(`<CheckRouter />`, () => {
+  beforeAll(() => {
+    jest.replaceProperty(config, 'bootData', {
+      // @ts-expect-error
+      user: {
+        orgRole: OrgRole.Editor,
+      },
+    });
+  });
```

Comment:
We have a global afterEach set up in `test/jest-setup.tsx` that clears all mocks so you can remove these, too.

This is my bad, I'm going to make a ticket to document how our testing framework works and what assumptions and things it does for you out of the box to make tests clearer and more predictable to write 😅